docs/validator/validator-faq.md

```diff
@@ -14,32 +14,55 @@ Fisherman (or validator) - nodes that provide security by verifying the validity
 
 ## How do I become a validator?
 
-If you already have an account with sufficient amount of funds.
+You need an account with sufficient amount of funds in the wallet.
 You can follow docs [here](local-setup/running-testnet.md) how to become validators
 
 Specific steps:
```

Comment:
this list would be easier to follow if it were numbered

1. first step
2. second step
3. etc.